src/app.ts

```diff
@@ -1,7 +1,30 @@
 import express from 'express'
 import path from 'path'
+import https from 'https'
 const app = express()
 const port = process.env.PORT || 3000
+
+// godaddy API_FROM_DB (JSON)
```

Comment:
This code will be removed when we'll have se tup a JSON DB

Reply:
Its set up here: https://github.com/garageScript/myProxy/pull/22/files

Hopefully @allopez7 's PR could be merged in soon :)